modules/jaeger-rn-new-features.adoc

```diff
@@ -0,0 +1,15 @@
+////
+Module included in the following assemblies:
+- rhbjaeger-release-notes.adoc
+////
+
+[id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 1.16
+
+////
+Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
+Result – If changed, describe the current user experience
+////
+
+This release of {ProductName} adds support for installing Jaeger as a standalone solution, rather than as a component of Red Hat OpenShift Service Mesh.
```

Comment:
This release adds support for stand alone in addition to with service mesh. Should it say `in addition to as a component of`.... RHOSM instead of rather than? Rather than suggests instead of, and it's both, right?

Reply:
No, you either have Jaeger by itself, or as part of the service mesh.